package.json

```diff
@@ -1,121 +1,119 @@
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "sf-husky-pre-commit",
-            "commit-msg": "sf-husky-commit-msg",
-            "pre-push": "sf-husky-pre-push"
-        }
-    },
-    "publishConfig": {
-        "access": "public"
+  "name": "salesforcedx",
+  "version": "50.14.4",
+  "description": "The Salesforce DX plugin aggregates all force topic commands into a single plugin that is bundled with the Salesforce CLI.",
+  "author": "Salesforce",
+  "bugs": "https://github.com/forcedotcom/cli/issues",
+  "files": [
+    "/lib",
+    "/messages",
+    "/oclif.manifest.json"
+  ],
+  "oclif": {
+    "commands": "./lib/commands",
+    "plugins": [
+      "salesforce-alm",
+      "@salesforce/plugin-templates",
+      "@salesforce/sfdx-plugin-lwc-test",
+      "@salesforce/plugin-custom-metadata",
+      "@salesforce/plugin-apex",
+      "@salesforce/plugin-schema",
+      "@salesforce/plugin-user",
+      "@salesforce/plugin-limits"
+    ]
+  },
+  "pinnedDependencies": [
+    "@salesforce/plugin-alias",
+    "@salesforce/plugin-custom-metadata",
+    "@salesforce/plugin-limits",
+    "@salesforce/plugin-schema",
+    "@salesforce/plugin-templates",
+    "@salesforce/plugin-user",
+    "@salesforce/sfdx-plugin-lwc-test",
+    "salesforce-alm"
+  ],
+  "dependencies": {
+    "@salesforce/plugin-alias": "1.1.5",
+    "@salesforce/plugin-apex": "0.1.4",
+    "@salesforce/plugin-custom-metadata": "1.0.11",
+    "@salesforce/plugin-limits": "1.0.2",
+    "@salesforce/plugin-schema": "1.0.3",
+    "@salesforce/plugin-templates": "50.4.0",
+    "@salesforce/plugin-user": "1.0.7",
+    "@salesforce/sfdx-plugin-lwc-test": "0.1.7",
+    "salesforce-alm": "50.14.0",
+    "salesforce-lightning-cli": "3.0.0"
+  },
+  "devDependencies": {
+    "@oclif/dev-cli": "^1",
+    "@salesforce/dev-config": "^2.0.0",
+    "@salesforce/dev-scripts": "0.6.2",
+    "@salesforce/prettier-config": "^0.0.1",
+    "@salesforce/ts-sinon": "1.1.3",
+    "@typescript-eslint/eslint-plugin": "^2.30.0",
+    "@typescript-eslint/parser": "^2.30.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "5.3.0",
+    "chai-fs": "1.0.0",
+    "chai-string": "1.2.0",
+    "cz-conventional-changelog": "^3.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "5.0.1",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-salesforce": "^0.1.0",
+    "eslint-config-salesforce-license": "^0.1.0",
+    "eslint-config-salesforce-typescript": "^0.1.0",
+    "eslint-plugin-header": "^3.0.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jsdoc": "^27.0.3",
+    "eslint-plugin-prettier": "^3.1.3",
+    "husky": "^4.2.5",
+    "mocha": "^7.2.0",
+    "nyc": "^15.1.0",
+    "prettier": "^2.0.5",
+    "pretty-quick": "^2.0.1",
+    "sinon": "^9.0.2",
+    "ts-node": "^8.10.2",
+    "typescript": "^3.9.3"
+  },
+  "config": {
+    "commitizen": {
+      "path": "cz-conventional-changelog"
     }
+  },
+  "engines": {
+    "node": ">=8.10.0"
```

Comment:
Shouldn't this be the v10 LTS?

Reply:
It is still v8 in sfdx-cli. I don't think we should require a new version until we update it there. Because that basically means that the latest salesforcedx isn't compatible with the latest cli. The old salesforcedx repo doesn't even have engine. I could just remove to keep it consistent with the current plugin?